### PR TITLE
ci: fix asana workflow for dependabot

### PR DESCRIPTION
**Pull request by bklebe**
#### Summary of changes
**Asana Ticket:** [[extra] fix Asana workflow in Prediction Analyzer](https://app.asana.com/0/584764604969369/1205953777846448/f)

Switches to the shared version of the Asana workflow. This should fix the problems with Dependabot and Asana.

#### Reviewer Checklist
- [ ] Meets ticket's acceptance criteria
- [ ] Any new or changed functions have typespecs
- [ ] Tests were added for any new functionality (don't just rely on Codecov)

### CODE REVIEW

File: .github/workflows/asana.yml [hunk 1]
```diff
@@ -1,38 +1,16 @@
...
-    steps:
-      - name: Create pull request attachments
-        uses: Asana/create-app-attachment-github-action@latest
-        id: postAttachment
-        with:
-          asana-secret: ${{ secrets.ASANA_GITHUB_INTEGRATION_SECRET }}
-      - name: Log output status
-        run: echo "Status is ${{ steps.postAttachment.outputs.status }}"
+  call-workflow:
+    uses: mbta/workflows/.github/workflows/asana.yml@main
```
Comment (meagharty) [inline .github/workflows/asana.yml:9]: It looks like there is also an asana workflow definition in this repo too, can we delete that file?

https://github.com/mbta/prediction_analyzer/blob/master/.github/workflows/asana.yml

File: .github/workflows/asana.yml [hunk 2]
```diff
@@ -1,38 +1,16 @@
...
+    with:
+      review-section: "Pending Review"
+      merged-section: "Merged / Not Deployed"
+      attach-pr: true
```
Comment (meagharty) [inline .github/workflows/asana.yml:13]: More of a deploy related question for this project: could we use `merged-section: "Done"` and `complete-on-merge: true`?

**Review by meagharty — APPROVED** [review]
LGTM! 

I added a deploy/process related question and a cleanup suggestion.

File: .github/workflows/asana.yml (hunk 1 above)
Comment (bklebe) [reply .github/workflows/asana.yml:9]: >It looks like there is also an asana workflow definition in this repo too, can we delete that file?
>
>https://github.com/mbta/prediction_analyzer/blob/master/

If you mean deleting `.github/workflows/asana.yml`, I believe this is the file I edited? Sorry, I'm a bit confused here.

File: .github/workflows/asana.yml (hunk 2 above)
Comment (bklebe) [reply .github/workflows/asana.yml:13]: I don't believe we have this written down anywhere (though we should) but we typically don't consider things done until they are deployed to prod.

File: .github/workflows/asana.yml (hunk 1 above)
Comment (meagharty) [reply .github/workflows/asana.yml:9]: Woops, ignore this. I was thinking there were two files in this repo, one for a re-usable workflow that wasn't being used anymore and one for this repo's workflow. There's only one for this repo's workflow.

**Pull request merged**